pallets/grant/src/mock.rs

```diff
@@ -1,20 +1,28 @@
 use crate as pallet_grant;
-use frame_support::parameter_types;
 use frame_system as system;
-use sp_core::H256;
+use sp_core::{
+	H256,
+	sr25519
+};
 use sp_runtime::{
 	testing::Header,
 	traits::{BlakeTwo256, IdentityLookup},
 	BuildStorage,
 	traits::ConstU32
 };
 use frame_support_test::TestRandomness;
-use frame_support::PalletId;
-
+use frame_support::{
+	PalletId,
+	once_cell::sync::Lazy,
+	parameter_types,
+};
 
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 type Block = frame_system::mocking::MockBlock<Test>;
+pub type AccountId = sr25519::Public;
+pub type Balance = u64;
 
+const EXISTENTIAL_DEPOSIT: u64 = 5;
```

Comment:
Where do we configure the existencial_deposit for runtime?

Reply:
line 249